PennMobile/Wrapped/WrappedContainerView.swift

```diff
@@ -0,0 +1,118 @@
+//
+//  WrappedContainerView.swift
+//  PennMobile
+//
+//  Created by Jonathan Melitski on 10/25/24.
+//
+
+import SwiftUI
+import Foundation
+
+struct WrappedContainerView: WrappedStage {
+    var activeState: WrappedExperienceState
+    
+    @Environment(\.scenePhase) var scenePhase
+    @EnvironmentObject var vm: WrappedExperienceViewModel
+    @EnvironmentObject var containerVM: WrappedContainerViewModel
+    let id: UUID = UUID()
+    var onFinish: (Bool) -> Void
+    
+    var body: some View {
+        TabView(selection: $containerVM.activeUnit) {
+            ForEach(containerVM.units) { unit in
+                WrappedUnitView(unit: unit, vm: containerVM)
+                    .tag(unit)
+            }
+        }
+        .allowsHitTesting(false)
+        .tabViewStyle(.page(indexDisplayMode: .never))
+        .ignoresSafeArea()
+        .background {
+            Color(red: 0.1, green: 0.1, blue: 0.1)
+        }
+        .overlay {
+            VStack {
+                HStack (spacing: 0) {
+                    ForEach(containerVM.units, id:\.self) { unit in
+                        GeometryReader { proxy in
+                            ZStack {
+                                Rectangle()
+                                    .size(width: CGFloat((Float(proxy.size.width))), height: 2)
+                                    .cornerRadius(1)
+                                    .foregroundStyle(.white.opacity(0.5))
+                                if unit.id <= containerVM.activeUnit.id {
+                                    Rectangle()
+                                        .size(width: unit.id < containerVM.activeUnit.id ? proxy.size.width :
+                                                CGFloat((Float(proxy.size.width) * Float(containerVM.activeUnitProgress))), height: 2)
+                                        .cornerRadius(1)
```

Comment:
You could also just use `RoundedRectangle()` since `.cornerRadius` is deprecated